test/test_notebook_document.py

```diff
@@ -1,10 +1,11 @@
 # Copyright 2021- Python Language Server Contributors.
 
-import os
 import time
 
 from unittest.mock import patch, call
 from test.fixtures import CALL_TIMEOUT_IN_SECONDS
+from test.test_utils import send_initialize_request, send_notebook_did_open
```

Comment:
```suggestion
from unittest.mock import patch, call

from test.test_utils import CALL_TIMEOUT_IN_SECONDS, send_initialize_request, send_notebook_did_open
```

* To separate standard library imports from testing ones.
* CALL_TIMEOUT_IN_SECONDS import was wrong.